### DISCUSSION TITLE
Error: Invalid aspect spec, should be WIDTH:HEIGHT or auto

### DISCUSSION THREAD

**Discussion by RodrigoDornelles** [SuperTux/supertux#3058] [General]
i cant run the game

```lisp
(supertux-config
  (profile 1)
  (show_fps #f)
  (show_player_pos #f)
  (show_controller #f)
  (developer #f)
  (confirmation_dialog #f)
  (pause_on_focusloss #t)
  (custom_mouse_cursor #t)
  (integrations
    (hide_editor_levelnames #f)
  )
  (editor_autosave_frequency 5)
  (transitions_enabled #t)
  (locale "")
  (repository_url "")
  (video
    (fullscreen #f)
    (video "auto")
    (vsync #t)
    (fullscreen_width 480)
    (fullscreen_height 320)
    (fullscreen_refresh_rate 0)
    (window_width 480)
    (window_height 320)
    (window_resizable #t)
    (aspect_width 600)
    (aspect_height 400)
    (magnification 0)
  )
  (audio
    (sound_enabled #t)
    (music_enabled #t)
    (sound_volume 100)
    (music_volume 50)
  )
 ...
```

**Comment by tobbi** [comment]
What version are you trying to run? Can you maybe try a nightly from http://download.supertux.org ?

**Comment by tobbi** [comment]
Or actually: How do you run supertux?

**Comment by RodrigoDornelles** [comment]
@tobbi i download from this website: <https://portmaster.games/detail.html?name=supertux> to use in my r36s console.

**Comment by tobbi** [comment]
I'm confused: You're mentioning SuperTuxKart, but this is a SuperTux config file, right?

**Reply by RodrigoDornelles** [reply]
> I'm confused: You're mentioning SuperTuxKart, but this is a SuperTux config file, right?



> I'm confused: You're mentioning SuperTuxKart, but this is a SuperTux config file, right?

wrong link, its is https://portmaster.games/detail.html?name=supertux, the supertuxkart works fine.

**Comment by tobbi** [comment]
The error message you're getting is one that gets raised when starting the game and providing the wrong parameters (see https://github.com/SuperTux/supertux/blob/f7518e7d2944538d32d36ba16f9be7e0749157de/src/supertux/command_line_arguments.cpp#L255) I suggest you contact the person porting the game: https://portmaster.games/profile.html?porter=Christian_Haitian

**Reply by RodrigoDornelles** [reply]
@christianhaitian

**Comment by christianhaitian** [comment]
@RodrigoDornelles Reach out to the PortMaster Team about this.  I don't maintain ports in PortMaster anymore.
